src/signals/incident-management/containers/IncidentDetail/components/EmailPreview/EmailPreview.tsx

```diff
@@ -1,26 +1,9 @@
 // SPDX-License-Identifier: MPL-2.0
-// Copyright (C) 2022 Gemeente Amsterdam, Vereniging van Nederlandse Gemeenten
-import { themeSpacing } from '@amsterdam/asc-ui'
-import styled from 'styled-components'
-
-import FormFooter from 'components/FormFooter'
-import { FORM_FOOTER_HEIGHT } from 'components/FormFooter/FormFooter'
+// Copyright (C) 2022 - 2023 Gemeente Amsterdam, Vereniging van Nederlandse Gemeenten
 import LoadingIndicator from 'components/LoadingIndicator'
 
-import ModalDialog from '../ModalDialog/ModalDialog'
-
-const StyledFormFooter = styled(FormFooter)`
-  .formFooterRow {
-    padding-left: ${themeSpacing(4)};
-  }
-`
-
-const StyledIframe = styled.iframe`
-  border: none;
-  padding: 0 0 ${FORM_FOOTER_HEIGHT}px ${themeSpacing(2)};
```

Comment:
How is it possible to just delete this?

Reply:
It's in the styled.ts.

Reply:
It's in the styled.ts

Reply:
@SireeKoolenWijkstra I don't see it anywhere in the styled connected to this component

Reply:
if you were looking for the FORM_FOOTER_HEIGHT, that was removed. It's back now.